### DISCUSSION TITLE
Still can't see the complete query collection after enabled.

### DISCUSSION THREAD

**Discussion by TeerapatSriboonsong** [planetscale/discussion#422] [Q&A]
As the title suggests, I want to see a clear log of what exactly my no-code platform queried into the database. 

But even after enabling the security for complete query collection option, the insights tab still show queries with placeholders just as before I enabled the option.

Is there sth I'm missing? 
Do I need to restart the database? 
Do I need to wait until the old 24 hour log history runs out so the new query logs don't get included into the previous placeholder logs? 
Should I've had switched the option even before creating the database, and now that it's already created I can't change that complete query option anymore?
Do I need to publish the production branch even if there's no visible change made to my branch?

I appreciate every help I can get rn  :)

**Comment by orware** [comment]
Hi @TeerapatSriboonsong,

Switching it on at anytime should be ok and shouldn't require any separate interactions from the user but I do believe there is a bit of a delay between when it is turned on and when you'll start to see the more detailed "Explain" links shown after clicking on a query listed within the table on the Insights page.

If you continue to experience issues with the feature, please reach out to us via https://support.planetscale.com/ that way we can more easily work through the difficulties together and you can share screenshots, etc. more easily.

**Reply by TeerapatSriboonsong** [reply]
The problem still persists somehow, I'll file a low priority request then.

Forgot to mentioned, I'm using a free plan. Don't think that matters for this problem but does it limit some functionalities?

Thank you for the reply :)
